packages/fast-permutator/__tests__/test-utils/index.js

```diff
@@ -1,5 +1,5 @@
-const get = require('lodash/get'),
-    permutator = require('../../src/index');
+const get = require('lodash-es').get;
```

Comment:
@janechu Is there a reason for the syntax change here back to dot notation? This ends up increasing our dependency on lodash-es from 9kb to 32kb.

Reply:
Yep...this resolves it.